capabilities/application-connectivity.md

```diff
@@ -0,0 +1,43 @@
+---
+displayName: Application Connectivity
+epicsLabels:
+  - area/application-connector
+---
+
+## Scope
+
+The Application Connectivity capability supports the extensibility and integration between applications in a cloud-native way using the best available solutions.
```

Comment:
For me, the scope is not clear. I would expect something that explains what problem Application Connectivity solves and how.